src/cursor.c

```diff
@@ -66,6 +66,19 @@ void cursor_key_to_move(int arrow, Cursor *cursor) {
     }
 }
 
+int cursor_opposite_dir(int arrow){
+    switch(arrow) {
+        case KEY_LEFT:
+            return KEY_RIGHT;
+        case KEY_RIGHT:
+            return KEY_LEFT;
+        case KEY_UP:
+            return KEY_DOWN;
+        case KEY_DOWN:
+            return KEY_UP;
+    }
+}
```

Comment:
When I compile I get a `control reaches end of non-void function` warning.

Reply:
Fun, I did not notice because it only happened the first time I compiled `cursor.c`
TIL: `make clean;and make frontend.out` before submitting a PR